src/core/ProgressBar/BaseProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useStyles } from './styles';
+import LinearProgress from '@material-ui/core/LinearProgress';
+
+interface ProgressBarProps {
+  progress: number;
+}
+const ProgressBar: React.FC<ProgressBarProps> = ({ progress }) => {
+  const classes = useStyles();
+  const getVariant = () => {
+    if (progress <= 40 && progress >= 0) {
+      return classes.low;
+    } else if (progress >= 41 && progress <= 80) {
+      return classes.warning;
+    } else {
+      return classes.good;
+    }
+  };
+  return (
+    <div>
+      <label className={classes.label}>Overall RR: {progress}</label>
```

Comment:
Replace this with a key-value pair, key is `label` prop and value would be `value` prop.